change/@microsoft-teams-js-2519cd0a-0e06-49b5-b79d-338d4042a1a3.json

```diff
@@ -1,6 +1,7 @@
 {
   "type": "patch",
   "packageName": "@microsoft/teams-js",
+  "comment": "update comment for initializePrivateApis explaining that this function needs to stay for backwards compatibility purposes",
```

Comment:
Can you please capitalize the first word "Update"?

Reply:
Is "initializePrivateApis" a function?  If so, can you update and format it like this: "`initializePrivateApis()` function"

Reply:
sure